app/static/css/_map.scss

```diff
@@ -1,9 +1,18 @@
+@use 'constants';
 @import 'npm:ol/ol.css';
 
 .map__container {
-    display: flex;
-    flex-flow: row wrap;
-    padding: 0 3rem;
+    display: block;
```

Comment:
Put the maps side by side on big screens and one on top of the other on small ones, with a gutter next to the map container so people can scroll past the map without accidentally having the map catch focus and making it zoom instead.